cypress/integration/examples/mainTest.spec.js

```diff
@@ -0,0 +1,18 @@
+describe("main test", () => {
```

Comment:
Let's maybe name this file mainTest.e2e-test.js so it can be easily filtered by regex

Reply:
done